lib/private/legacy/OC_Image.php

```diff
@@ -415,7 +415,7 @@ public function __toString() {
 	 * @return int|null
 	 */
 	protected function getJpegQuality() {
-		$quality = $this->config->getAppValue('preview', 'jpeg_quality', 90);
+		$quality = $this->config->getAppValue('preview', 'jpeg_quality', 80);
```

Comment:
```suggestion
		$quality = $this->config->getAppValue('preview', 'jpeg_quality', '80');
```
`OCP\IConfig::getAppValue` expects string and the value is already cast to int below https://github.com/nextcloud/server/pull/35203/files#diff-117d672dbebb895f2a089afd40234274f83152f5d9cd8a12f3e4cd5d61f0417bR420